JS/Lesson34/goksel/recipe-app/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,29 @@
+import "./styles.css";
+import { Link } from "react-router-dom";
+
+export const Navbar = ({ isSignedIn, setIsSignedIn }) => {
+
+    const handleClick = () => {
+        setIsSignedIn(!isSignedIn);
+    }
+    return (
+        <nav className="navbar">
+            <ul>
+                {isSignedIn ?
+                    <div className="navbar-btns">
+                        <li><Link to="/"><i class="fa-solid fa-house"></i>Home Page</Link></li>
+                        <li><Link to="user/register"><i class="fas fa-user"></i>Login</Link></li>
```

Comment:
Register and log in should be different pages.

Reply:
Done.